README.md

```diff
@@ -138,6 +138,37 @@ Then try running `npm install` again.
 
 Please see [build instructions](https://github.com/deltachat/deltachat-core#building-your-own-libdeltachatso) for additional information.
 
+### Logging
+
+#### Logging Options
+Debug messages are disabled by default, enable them with the `--log-debug` flag.
+
+flag               | effect
+-------------------|---------------------------------------------
+`--log-debug`      | Log debug messages
+`--log-to-console` | Output the log to stout / chrome dev console
+
+
+#### Logfile
+
+The Logfiles can be found in:
+```
+Linux: ~/.config/DeltaChat/logs/
+Mac: "~/Library/Application Support/DeltaChat/logs"
```

Comment:
Is the path for mac correct?

Reply:
Can't test it as i don't have a mac around. I guess we need to check it out at bb.

Reply:
It's correct https://github.com/LinusU/node-application-config-path/blob/master/index.js#L5